src/css/style.css

```diff
@@ -1,4 +1,108 @@
-/* Your custom styles go here! */
+:root {
+    --grey-lightest: #f7f9fa;
+    --blue-light: rgba(43, 63, 86, 0.81);
+    --green: #72cc18;
+    --green-light: rgba(114, 204, 24, 0.68);
+    --orange: orange;
+    --orange-dark: darkorange;
+}
+
+html, body {
+    margin: 0;
+    height: 100%;
+    font-family: "Helvetica Neue", Sans-Serif, sans-serif;
+    background: var(--blue-light);
+    color: var(--grey-lightest);
+    display: flex;
+    flex-direction: column;
+}
+
+body a {
+    color: var(--orange);
+}
+
+body a:hover {
+    color: var(--orange-dark);
+}
+
+ul {
+    list-style-type: square;
+    list-style-position: inside;
+    overflow-y: scroll;
```

Comment:
There's an overflow-y on the #presence element already so you can remove this one. (It's displaying two scroll bars currently)
![Screen Shot 2023-08-09 at 11 53 40 AM](https://github.com/daily-demos/daily-prejoin-presence/assets/18265845/80706c48-e7ae-4fec-a59d-2cf751a2758d)